prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -495,6 +497,19 @@ private async Task CreateSelfDeclarationDocuments(int enrolleeId, ICollection<Se
                     });
                 }
             }
+
+            foreach (var document in currentSelfDeclarationDocuments)
+            {
+                if (!newDeclarations.Select(nd => nd.SelfDeclarationTypeCode).Contains(document.SelfDeclarationTypeCode))
```

Comment:
rather than doing `.Select.Contains` you can just do ` if (!newDeclarations.Any(newDocument => newDocument.SelfDeclarationTypeCode == oldDocument.SelfDeclarationTypeCode))`